### ISSUE TITLE
[Digitizer] font definition for size/weight/fontfamily is not used

### ISSUE THREAD

**Issue by astroidex** [mapbender/mapbender#1308]
**Describe the bug**
Font definition for labels is not displayed for  size/weight/fontfamily

You can define the color, font weight, font family for the result style. But it has no effect. 

**To Reproduce**
1. Define a style like this for you result object:
2. If you move an object it should get the definied labeling. The color is used. But not the size/weight/fontFamily

```
    unsaved:
      strokeWidth: 5
      strokeColor: '#ff0000'
      label: 'Neu - bitte speichern'
      fontFamily: 'Arial, Courier New, monospace'      
      fontColor: red
      fontSize: 38
      fontWeight: Bold
```

**Expected behavior**
Red labeling in size 38 weight and font family  'Arial, Courier New, monospace' for modified objects

**Versions (please complete the following information if it helps to narrow down the bug):**
 - Mapbender: 3.2.3

**Comment by astroidex** [comment]
Here is a PR for the bug: https://github.com/mapbender/ol4-extensions/pull/2
I only changed the ordering in the font definition

**Comment by thzanger** [comment]
Dear @astroidex,
thank you for this information. I forwarded the information into our internal ticket system.
Regards

**Comment by thzanger** [comment]
Dear @astroidex, the issue has been solved in the latest version (3.2.4). This issue will thus be closed.

**Comment by astroidex** [comment]
tested with 3.2.4: works fine
